lib/index.js

```diff
@@ -189,6 +188,27 @@ module.exports.render = function(options, cb) {
     };
   }
 
+  if (options.functions) {
+    Object.keys(options.functions).forEach(function(signature) {
+      var callback = options.functions[signature];
+
+      options.functions[signature] = function() {
+        var args = Array.prototype.slice.call(arguments),
+            bridge = args.pop();
+
+        function done(data) {
+          bridge.success(data);
+        }
+
+        var result = callback.apply(this, args.concat(done));
```

Comment:
@Jakobo Yes this makes complete sense. Ultimately I can see that the context could be used to call built-in sass functions, or functions defined through the sass files.